src/rez/rex.py

```diff
@@ -684,6 +686,59 @@ def get_key_token(self, key):
         # here because the API requires it.
         return "${%s}" % key
 
+    def adjust_env_for_platform(self, env):
+        """ Make required platform-specific adjustments to env.
+        """
+        if sys.platform.startswith('win'):
+            env = self._add_systemroot_to_env_win32(env)
```

Comment:
Same as above, in-place rather than copy

Reply:
fixed!